crates/ruff/src/rules/pycodestyle/rules/logical_lines/mod.rs

```diff
@@ -100,9 +101,25 @@ impl<'a> LogicalLines<'a> {
                 TokenKind::Rbrace | TokenKind::Rpar | TokenKind::Rsqb => {
                     parens -= 1;
                 }
-                TokenKind::Newline | TokenKind::NonLogicalNewline | TokenKind::Comment
-                    if parens == 0 =>
-                {
+                TokenKind::Comment if parens == 0 => {
```

Comment:
Do we still need the extra casing after the changes in RustPython land?

Reply:
We definitely need _some_ change, I _think_ we want to just remove comments entirely here since end-of-line is now always indicated by a newline token?